newIDE/app/src/UI/Theme/DefaultTheme/index.js

```diff
@@ -125,6 +125,11 @@ const muiTheme = createMuiTheme({
       },
     },
     MuiInput: {
+      input: {
+        padding: 0,
+        paddingBottom: 1,
+        zIndex: 1,
```

Comment:
I'm always a bit worried of putting zIndex... this can sometimes create hard to debug visual issues with other controls and force the browser/GPU to create a new texture in memory for rendering the element.
What's the result if we don't put it? :)

Reply:
Without / With
![image](https://user-images.githubusercontent.com/1670670/92946490-087d8f80-f457-11ea-8a97-b5a04e69eb13.png)
Let's go without, with a bit more padding on bottom 👍